lib/platform.py

```diff
@@ -25,7 +25,13 @@ def load_caps():
     )
   )
   if os.path.exists(capsfile):
-    execfile(capsfile, namespace)
+    global_namespace = {
+      "__file__": capsfile,
+      "__name__": "__main__",
+    }
+    with open(capsfile, 'rb') as f:
+      exec(f.read(), global_namespace, namespace)
```

Comment:
Just use `exec(f.read())`... I don't think setting globals and locals is necessary.

Reply:
Wait, I guess you will need `exec(f.read(), namespace)` instead.

Reply:
did the change